authN-authZ/README.md

```diff
@@ -223,48 +223,48 @@ The user management is done via Keycloak and the configuration steps look like t
+   ![create clent 1](./docs/create_client_1.png)
 
-<img src="./docs/create_client_2.png" width="600" height="300">
+   ![create clent 2](./docs/create_client_2.png)
 
-<img src="./docs/create_client_3.png" width="600" height="300">
+   ![create clent 3](./docs/create_client_3.png)
 
 3. Access the dedicated mappers pane by clicking `<your client's id>-dedicated`, located under Assigned client scope to configure a new `Audience` mapper with name `aud-mapper-<your client's id>`. And include Audience in your client with `ID token` and `access token` set to `On`.
 
-<img src="./docs/add_mapper.png" width="600" height="300">
+   ![add mapper](./docs/add_mapper.png)
 
 4. Create new roles `user` and `viewer` by navigating to `<your client's id> -> Roles`.
 
 5. Create a new user name as `mary` and another user as `bob` with `Email verified` set to `On`. Set passwords for both users (set 'Temporary' to 'Off').
 
-<img src="./docs/create_user.png" width="600" height="300">
+   ![create user](./docs/create_user.png)
 
 6. Create a new Client Scope with the name `groups` in Keycloak with `Include in Token Scope` set as `On`. Include a mapper of type `Group Membership` and set the `Token Claim Name` to `groups`. If the "Full group path" option is selected, you need to include a "/" separator in the group names defined in the --allowed-group option of OAuth2 Proxy. Example: "/groupname". After creating the Client Scope named `groups` you will need to attach it to your client. Go to Clients and find `<your client's id> -> Client scopes` and add client scope and select `groups` and choose `Optional` and you should now have a client that maps group memberships into the JWT tokens so that Oauth2 Proxy may evaluate them.
 
-<img src="./docs/add_group_scope.png" width="600" height="300">
+   ![add group scope](./docs/add_group_scope.png)
 
-<img src="./docs/attach_group_scope.png" width="600" height="300">
+   ![attach group scope](./docs/attach_group_scope.png)
 
 7. Create two groups `user` and `viewer` by navigating to Groups -> Create group. Assign role `user` to group `user` and role `viewer` to group `viewer` and add user `mary` as a member of group `user` and `bob` as a member of group `viewer`.
 
-Next, we will install the oauth2-proxy and configure the OIDC information.
-
-```bash
-# set values for realm name, client name and ingress port
-# then deploy oauth2-proxy
-export REALM=<YOUR_REALM_NAME>
-export CLIENT=<YOUR_CLIENT_NAME>
-export CLIENT_SECRET=<YOUR_CLIENT_SECRET>
-# generate cookie secret
-# Using bash here. More methods found here:
-# https://oauth2-proxy.github.io/oauth2-proxy/configuration/overview#generating-a-cookie-secret
-export COOKIE_SECRET=$(dd if=/dev/urandom bs=32 count=1 2>/dev/null | base64 | tr -d -- '\n' | tr -- '+/' '-_' ; echo)
-envsubst < $(pwd)/oauth2_install.yaml | kubectl apply -f -
-```
+   Next, we will install the oauth2-proxy and configure the OIDC information.
```

Comment:
Can we remove the blanks here? As setting up oauth2-proxy is not part of the steps of keycloak configuration.